src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -64,8 +63,7 @@
  */
 
 @RunWith(SpringRunner.class)
-@WebMvcTest(value = NetworkConversionController.class, properties = {"spring.data.elasticsearch.enabled=true"})
-@ContextConfiguration(classes = {NetworkConversionApplication.class, NetworkConversionService.class, EmbeddedElasticsearch.class})
+@WebMvcTest(value = NetworkConversionController.class)
```

Comment:
Why did you completely remove line `@ContextConfiguration(classes = {NetworkConversionApplication.class, NetworkConversionService.class, EmbeddedElasticsearch.class})` ?

Reply:
No need to indicate the context configuration (just a cleaning)